<h1 align="center"> Anastasia Zolotar :pineapple: </h1>

[![Linkedin Badge](https://img.shields.io/badge/-ananaszol-blue?style=flat-square&logo=Linkedin&logoColor=white&link=[https://www.linkedin.com/in/tanejasaksham/](https://www.linkedin.com/in/ananas-zol/))](https://www.linkedin.com/in/ananas-zol/) 
[![GitHub followers](https://img.shields.io/github/followers/ananas-des?label=Follow&style=social)](https://github.com/ananas-des/?tab=follow) 
[![Gmail Badge](https://img.shields.io/badge/-zolotar.ananas@gmail.com-c14438?style=flat-square&logo=Gmail&logoColor=white&link=mailto:zolotar.ananas@gmail.com)](mailto:zolotar.ananas@gmail.com) 
[![Telegram Badge](https://img.shields.io/badge/-@ananasdes-blue?style=flat-square&logo=Telegram&logoColor=white&link=[https://www.linkedin.com/in/tanejasaksham/](https://t.me/ananas_des))](https://t.me/ananas_des)

<img src="https://media.giphy.com/media/cBSxWiay0sbNkEElG7/giphy.gif" width="200"> [![Typing SVG](https://readme-typing-svg.demolab.com?font=Rubic&size=28&duration=3500&pause=500&color=D0072CE9&vCenter=true&multiline=true&width=435&height=100&lines=Bioinformatician;Molecular+biologist)](https://git.io/typing-svg) </br>
[Bioinformatics Institute](https://bioinf.me/en), Bioinformatics for Biologists </br>
[Lomonosov MSU](https://www.msu.ru/en/), BSc and MSc in Biology </br>

## My research experience üêÄ

<details><summary>
<b>Bioinformatic Institute project</b>: SNP grouping and evaluation of epistatic interaction for phenotypic traits in agricultural plants
</summary><br> 
 
We aimed to find a software and workflow for grouping single nucleotide polymorphisms (SNPs) based on their association with a quantitative trait and performing epistatic analysis. To achieve this goal, we utilized free-access data on the alanine content of soybeans *Glycine max*, as well as a dataset with some complex phenotypic trait (Nova Plant commercial
property). The results allowed us to construct interaction graphs of soybean genes which will help the Nova Plant company to work on crop improvement in the future.<br>

Also, during intensive one-year retraining program I worked on wide bioinformatics tasks using open-access data, such as:<br>

1) variant calling of *Escherichia coli* WGS and deep sequencing data;
2) *de novo* assembly of its genome;
3) performing tardigrade *Ramazzottius varieornatus* genome annotation and protein function prediction;
4) performing genotyping and SNP annotation of human 23andMe project data;
5) analyzing differential gene expression of *Saccharomyces cerevisiae* after 30 minutes of fermentation;
6) analyzing ancient metagenomes examining human dental calculus;
7) and, just a little, immune repertoire annotation based on T-cell population of a relatively healthy donor.
</details>

[Here](https://drive.google.com/drive/folders/1OoKH-iIfSdFm2jrxx-7S5D8Ghl-K16fw?usp=sharing), there are examples of my bioinformatics mini projects.

<details><summary>
<b>MSc thesis</b>: Single-cell transcriptomics of a trained command neuron of <i>Helix lucorum</i>: SNP grouping and evaluation of epistatic interaction for phenotypic traits in agricultural plants
</summary><br>
 
I conducted fear conditioning experiments with aversive stimulus on terrestrial gastropod *Helix lucorum*, extract parietal Pa2 and Pa3 neurons, which are known to orchestrate the stimulus avoidance, prepared cDNA libraries, and performed scRNA-Seq of these neurons. Also I took part in a project on studying the splicing and differential gene expression after picrotoxin (cocculine) activation of rat primary hippocampal culture using full-length mRNA sequencing on MinION Oxford Nanopore sequencer.
</details>

<details><summary>
<b>BSc thesis</b>: Effect of EF-hand proteins semiconservative amino acid cluster on structural and functional human calmodulin properties
</summary><br>
 
I conducted alanine screening mutagenesis of the conservative and semi-conservative amino acids in human calmodulin EF-hand motif to define crucial ones for maintaining calmodulin structure and affecting its function.

</details>

## My programming skills üíª

 - **Python** (numpy, pandas, matplotlib, seaborn, SciPy, statsmodels, scikit-learn, PyTorch, requests, BeautifulSoup, Biopython, ete3)

![Python](https://img.shields.io/badge/python-3670A0?style=for-the-badge&logo=python&logoColor=ffdd54)
![Jupyter Notebook](https://img.shields.io/badge/jupyter-%23FA0F00.svg?style=for-the-badge&logo=jupyter&logoColor=white)
![Colab](https://img.shields.io/badge/Colab-F9AB00?style=for-the-badge&logo=googlecolab&color=525252)
![NumPy](https://img.shields.io/badge/numpy-%23013243.svg?style=for-the-badge&logo=numpy&logoColor=white) 
![Pandas](https://img.shields.io/badge/pandas-%23150458.svg?style=for-the-badge&logo=pandas&logoColor=white) 
![Matplotlib](https://img.shields.io/badge/Matplotlib-%23ffffff.svg?style=for-the-badge&logo=Matplotlib&logoColor=black)
![scikit-learn](https://img.shields.io/badge/scikit--learn-%23F7931E.svg?style=for-the-badge&logo=scikit-learn&logoColor=white) 
![PyTorch](https://img.shields.io/badge/PyTorch-%23EE4C2C.svg?style=for-the-badge&logo=PyTorch&logoColor=white)
![SciPy](https://img.shields.io/badge/SciPy-%230C55A5.svg?style=for-the-badge&logo=scipy&logoColor=%white)

- **R** (dplyr, ggplot2, tidyverse, tidyr, ape, ggtree)

![R](https://img.shields.io/badge/r-%23276DC3.svg?style=for-the-badge&logo=r&logoColor=white)
![RStudio](https://img.shields.io/badge/RStudio-4285F4?style=for-the-badge&logo=rstudio&logoColor=white)

- **Linux**

![Linux](https://img.shields.io/badge/Linux-FCC624?style=for-the-badge&logo=linux&logoColor=black)
![GNU bash](https://img.shields.io/badge/GNU%20Bash-4EAA25?style=for-the-badge&logo=GNU%20Bash&logoColor=white)
![Shell Script](https://img.shields.io/badge/shell_script-%23121011.svg?style=for-the-badge&logo=gnu-bash&logoColor=white)
![Docker](https://img.shields.io/badge/docker-%230db7ed.svg?style=for-the-badge&logo=docker&logoColor=white)

- **Others**

![LaTeX](https://img.shields.io/badge/latex-%23008080.svg?style=for-the-badge&logo=latex&logoColor=white)
![Overleaf](https://img.shields.io/badge/Overleaf-47A141?style=for-the-badge&logo=Overleaf&logoColor=white)
![Markdown](https://img.shields.io/badge/markdown-%23000000.svg?style=for-the-badge&logo=markdown&logoColor=white)
![Notion](https://img.shields.io/badge/Notion-000000?style=for-the-badge&logo=notion&logoColor=white)

![Anastasia Zolotar's GitHub stats](https://github-readme-stats.vercel.app/api?username=ananas-des&show_icons=true&theme=dracula)
